### PR TITLE
#8 Speakers-section

### PR DESCRIPTION
**Pull request by Bhupendra-Gehlot1**
Added file9, GitHub, Linkedin, Twitter, and Facebook icons in src/assets/images
Created section 4 folder containing Section4.js and Section4.css
Added Section4 Component in Homepage.js
Added href=#speakers in Navbar Component
Created speakerCard folder in components containing speakerCard.js and speakerCard.css

### CODE REVIEW

File: src/sections/section4/Section4.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.codeutsava__section4 {
+    padding-top: 2rem;
+    padding-bottom: 2rem;   
+    font-family: 'Poppins';
```
Comment (sahil9001) [inline src/sections/section4/Section4.css:4]: Different fonts are being used in different sections of the card, so please refer to figma design as they are using "Roboto" for paragraphs.

File: src/components/speakerCard/SpeakerCard.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    padding-top: 0.5rem;
+}
+
+.codeutsava__section4-speaker-menu p{
+    font-weight: 300;
+    font-size: 15px;
+    line-height: 18px;
+    text-align: center;
+    padding: 1rem 0;
+}
```
Comment (sahil9001) [inline src/components/speakerCard/SpeakerCard.css:43]: Use roboto font for paragraphs

**Review by sahil9001 — CHANGES_REQUESTED** [review]

**Comment by prakhar-asaiya** [comment]
Put another row of cards so we can see how vertical positioning of cards looks like.

File: src/components/speakerCard/SpeakerCard.css (hunk 2 above)
Comment (Bhupendra-Gehlot1) [reply src/components/speakerCard/SpeakerCard.css:44]: Added robot font in paragraphs in speaker card and two more cards to show vertical positioning of cards.

**Comment by sahil9001** [comment]
By writing 'Roboto' in the font family it doesn't gets added, you need to import it in the css refer this [here](https://fonts.google.com/specimen/Roboto)

**Comment by prakhar-asaiya** [comment]
Firstly import roboto from google fonts and add it to css file. Then it'll start working.

**Comment by Bhupendra-Gehlot1** [comment]
> By writing 'Roboto' in the font family it isn't added, you need to import it in the css refer this [here](https://fonts.google.com/specimen/Roboto)



> By writing 'Roboto' in the font family it isn't added, you need to import it in the css refer this [here](https://fonts.google.com/specimen/Roboto)

**Comment by Bhupendra-Gehlot1** [comment]
Imported and added Roboto font please Check... 
Sorry closed pr by mistake

File: src/components/navbar/Navbar.js [hunk 3]
```diff
@@ -18,10 +18,10 @@ export default class Navbar extends Component {
                 </div></a>  
             <a href='#events'><div className='codeutsava__navbar-menu-heading'>
                     Events
-                </div></a>  
-                <div className='codeutsava__navbar-menu-heading'>
-                    Sponsors
-                </div>
+                </div></a>
```
Comment (sahil9001) [inline src/components/navbar/Navbar.js:21]: Instead of removing the "sponsors" section, add another section named "speakers" after the sponsors one.

**Review by sahil9001 — CHANGES_REQUESTED** [review]

File: src/components/navbar/Navbar.js (hunk 3 above)
Comment (Bhupendra-Gehlot1) [reply src/components/navbar/Navbar.js:21]: added both "sponsors" and "speakers sections. added "speakers" section after the "sponsors" section in navbar .

**Review by prakhar-asaiya — APPROVED** [review]

**Pull request merged**